### PR TITLE
volunteer individual activity

### PR DESCRIPTION
**Pull request by ashatat**
#63

#### Issues this PR relates to

---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist
Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)
- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

**Review by dupreesi — CHANGES_REQUESTED** [review]
AMAZING JOB! the only bits I could find are:
- [ ] on individual activity page the font style of "Please note, clicking the button will open in a new tab" doesn't match wireframes
- [ ] Skill complete page doesn't match wireframes
Bildschirmfoto 2021-02-02 um 12 04 04
- [ ] We're rolling page: the blue number in the circle should show the total amount of points (not the old one), so in the screen below it should show 28 (like in the text below)
Bildschirmfoto 2021-02-02 um 12 06 04
- [ ] decrease padding on desktop for recommended activities section
Bildschirmfoto 2021-02-02 um 12 07 34
- [ ] when logged out and completing activity the success page no recommended skills are loading
Bildschirmfoto 2021-02-02 um 12 09 59

File: client/src/pages/HqDashboard/index.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ const { Row, Col } = Grid;
 function HqDashboard() {
   const [stats, setStats] = useState({});
   const [loading, setLoading] = useState(false);
-  const [err, setErr] = useState('');
+  // const [err, setErr] = useState('');
```
[the remark this answers is not in the document]
Comment (RamyAlshurafa) [reply client/src/pages/HqDashboard/index.js:17]: I think if we leave it for now and we uncomment them in future to show them to users, better than deleting them and then write them again when they needed

File: client/src/constants/nav-routes.js [hunk 2]
```diff
@@ -23,8 +24,9 @@ const VOLUNTEER = {
   DASHBOARD: '/dashboard',
   ENTER_POSTCODE: '/postcode',
   RESULTS: '/result',
-  INDIVIDUAL_SKILL: `/skills/:id`,
-  INDIVIDUAL_ACTIVITY: `/activities/:id`,
+  SETTINGS: '/settings',
+  // INDIVIDUAL_SKILL: `/skills/:id`,
```
[the remark this answers is not in the document]
Comment (RamyAlshurafa) [reply client/src/constants/nav-routes.js:28]: done

**Comment by RamyAlshurafa** [comment]
> AMAZING JOB! the only bits I could find are:
> 
> * [ ]  on individual activity page the font style of "Please note, clicking the button will open in a new tab" doesn't match wireframes

Can you check again (its BodyB in code which is 18px same as Figma) 

> * [ ]  Skill complete page doesn't match wireframes
> 

Just agreed with @thejoefriel  to fix this in his feedback PR


> Bildschirmfoto 2021-02-02 um 12 04 04
> 
> - [ ] We're rolling page: the blue number in the circle should show the total amount of points (not the old one), so in the screen below it should show 28 (like in the text below)
> Bildschirmfoto 2021-02-02 um 12 06 04
> 
> - [x] decrease padding on desktop for recommended activities section
> Bildschirmfoto 2021-02-02 um 12 07 34
> 
> - [ ] when logged out and completing activity the success page no recommended skills are loading
> Bildschirmfoto 2021-02-02 um 12 09 59

Agreed with @thejoefriel to fix it in the feedback PR

**Review by dupreesi — APPROVED** [review]

**Pull request merged**